src/Module/Contact.php

```diff
@@ -891,31 +901,31 @@ public static function getTabsHTML($a, $contact, $active_tab)
 		];
 
 		// Show this tab only if there is visible friend list
-		$x = Model\GContact::countAllFriends(local_user(), $contact['id']);
```

Comment:
Please keep using the `GContact` methods or include the extra condition they feature in this module.